frontend/public/components/masthead-toolbar.jsx

```diff
@@ -592,5 +598,10 @@ const mastheadToolbarStateToProps = ({ UI }) => ({
 });
 
 export const MastheadToolbar = connect(mastheadToolbarStateToProps)(
-  connectToFlags(FLAGS.AUTH_ENABLED, FLAGS.OPENSHIFT, FLAGS.CLUSTER_VERSION)(MastheadToolbar_),
+  connectToFlags(
+    FLAGS.AUTH_ENABLED,
+    FLAGS.OPENSHIFT,
+    FLAGS.CLUSTER_VERSION,
+    FLAGS.CONSOLE_CLI_DOWNLOAD,
```

Comment:
nit: I tend to alphabetize lists when they start to get large and order doesn't matter